3rdparty/ndsrvp/include/imgproc.hpp

```diff
@@ -1,20 +1,28 @@
 // This file is part of OpenCV project.
 // It is subject to the license terms in the LICENSE file found in the top-level directory
-// of this distribution and at http://opencv.org/license.html.	
+// of this distribution and at http://opencv.org/license.html.
 
 #ifndef OPENCV_NDSRVP_IMGPROC_HPP
 #define OPENCV_NDSRVP_IMGPROC_HPP
 
 namespace cv {
 
-// ################ remap ################
-
-void remap(InputArray _src, OutputArray _dst,
-    InputArray _map1, InputArray _map2,
-    int interpolation, int borderType, const Scalar& borderValue);
-
 namespace ndsrvp {
 
+enum InterpolationFlags{
+    INTER_NEAREST        = 0,
```

Comment:
It would be better to add missing values to the `imgproc/hal/interface.h` and use them in HAL.

Reply:
Done.